packages/components/src/Menu/MenuItem.test.tsx

```diff
@@ -47,3 +49,20 @@ test('MenuItem - current', () => {
     </MenuItem>
   )
 })
+
+test('MenuItem - artwork', () => {
+  assertSnapshot(
+    <MenuItem
+      artwork={
+        <svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path
+            d="M12 2C6.48 2 2 6.48 2 12C2 17.52 6.48 22 12 22C17.52 22 22 17.52 22 12C22 6.48 17.52 2 12 2ZM13 17H11V15H13V17ZM13 13H11V7H13V13Z"
+            fill="#7FFFD4"
+          />
+        </svg>
+      }
+    >
+      Artwork
+    </MenuItem>
+  )
+})
```

Comment:
Let's avoid adding a snapshot. Instead let's make a SVG we can find by title and then look for that. PSUEDO CODE:
```suggestion
test('MenuItem - artwork', () => {
    renderWithTheme(
    <MenuItem
      artwork={
        <svg xmlns="http://www.w3.org/2000/svg">
           <title>custom artwork</title>
        </svg>
      }
    >
      Artwork
    </MenuItem>
  )
  
  expect(screen.findByTitle('custom artwork')).toBeVisible()
})
```